From ce9a2f67cbe161a8fc1a8671ae736dc886feb66f Mon Sep 17 00:00:00 2001
From: Michael McAuliffe <michael.e.mcauliffe@gmail.com>
Date: Wed, 11 Jan 2023 20:21:58 -0800
Subject: [PATCH] Remove native arch for cross-compilation

---
 Makefile | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/Makefile b/Makefile
index 3a350d4..5a9bfa5 100644
--- a/Makefile
+++ b/Makefile
@@ -9,14 +9,7 @@ TESTS = $(wildcard test/sql/*.sql)
 REGRESS = $(patsubst test/sql/%.sql,%,$(TESTS))
 REGRESS_OPTS = --inputdir=test --load-extension=vector
 
-OPTFLAGS = -march=native
-
-# Mac ARM doesn't support -march=native
-ifeq ($(shell uname -s), Darwin)
-	ifeq ($(shell uname -p), arm)
-		OPTFLAGS =
-	endif
-endif
+OPTFLAGS =
 
 # For auto-vectorization:
 # - GCC (needs -ftree-vectorize OR -O3) - https://gcc.gnu.org/projects/tree-ssa/vectorization.html
@@ -36,9 +29,6 @@ sql/$(EXTENSION)--$(EXTVERSION).sql: sql/$(EXTENSION).sql
 
 EXTRA_CLEAN = sql/$(EXTENSION)--$(EXTVERSION).sql
 
-PG_CONFIG ?= pg_config
-PGXS := $(shell $(PG_CONFIG) --pgxs)
-include $(PGXS)
 
 # for Mac
 ifeq ($(PROVE),)
-- 
2.38.1

